app/components/post_textbox/post_textbox.js

```diff
@@ -451,6 +458,35 @@ export default class PostTextbox extends PureComponent {
         });
     };
 
+    onCloseChannelPress = () => {
+        const {defaultChannel, channelId} = this.props;
+        const {setChannelDisplayName, setChannelLoading} = this.props.actions;
+        setChannelLoading(true);
+        setChannelDisplayName(defaultChannel.display_name);
+        EventEmitter.emit('switch_channel', defaultChannel, channelId);
```

Comment:
I don't think this cover the case where the postTextBox is being shown in the Thread view

Reply:
Do you mean the permalink view? I don't understand.